platform/jobs/pipelines/edxPlatformPipelinePr.groovy

```diff
@@ -79,18 +107,18 @@ Map publicQualityJobConfig = [
     pythonVersion: '3.8',
 ]
 
-Map django30QualityJobConfig = [
+Map django32QualityJobConfig = [
     open: true,
-    jobName: 'edx-platform-django-3.0-quality-pipeline-pr',
+    jobName: 'edx-platform-django-3.2-quality-pipeline-pr',
     repoName: 'edx-platform',
     whitelistBranchRegex: /^((?!open-release\/).)*$/,
-    context: 'jenkins/django-3.0/quality',
+    context: 'jenkins/django-3.2/quality',
     onlyTriggerPhrase: true,
-    triggerPhrase: /.*jenkins\W+run\W+django30\W+quality.*/,
+    triggerPhrase: /.*jenkins\W+run\W+django31\W+quality.*/,
```

Comment:
Was it supposed to be `django32` instead?

Reply:
yes